tests/features/joinup_seo/news.feature

```diff
@@ -0,0 +1,53 @@
+@api
+Feature: As an owner of the website
+  in order for my news to be better visible on the web
+  I need proper metatag to be encapsulated in the html code.
```

Comment:
Feature should have a title, now the first line of the user story is used as the title of the feature. A good title would be something like "SEO for news articles"